### ISSUE TITLE
Update to Easy Forms for MailChimp

### ISSUE THREAD

**Issue by Nelmedia** [yikesinc/yikes-inc-easy-mailchimp-extender#463]
Hi,

Just updated to the latest version of Easy Forms for MailChimp (6.0.4.7) and finally I was able to connect to my account... But now, almost every options of the plugin are only grey empty pages... 

Testing site: http://test.tremblaylecuyer.com/Lib 

Using WP 4.4.2

**Comment by EvanHerman** [comment]
Hey Nelmedia,

I am going to push out an update today. I am in the process of completely removing the MailChimp php wrapper, which is the culprit here. The php wrapper submits requests via CURL - and some weird things have been happening for a few users on shared hosts.

I'll post back here when I get that patch out.


Thanks!
Evan

**Comment by EvanHerman** [comment]
Hey Nelmedia,

I've just pushed out v6.0.5 to the repository. Give things an update and let me know if you are running into any issues.


Thanks!
Evan

**Comment by TheTracyL** [comment]
Hi Nelmedia,

Just checking in, let us know if that worked for you.

Thank you!
-Tracy

**Comment by Nelmedia** [comment]
Yes. As mentioned, with the first update it created other problems, but with the update that was released Saturday, then all is working fine and I was able to add the form to my web site.

 

Thank you!

 

Nelson

 

De : Tracy Levesque [mailto:notifications@github.com] 
Envoyé : 7 mars 2016 11:29
À : yikesinc/yikes-inc-easy-mailchimp-extender
Cc : Nelmedia
Objet : Re: [yikes-inc-easy-mailchimp-extender] Update to Easy Forms for MailChimp (#463)

 

Hi Nelmedia,

Just checking in, let us know if that worked for you.

Thank you!
-Tracy
